cfbs/args.py

```diff
@@ -62,4 +62,9 @@ def _get_arg_parser():
         help="Keep order of items in the JSON in 'cfbs pretty'",
         action="store_true",
     )
+    parser.add_argument(
+        "--git",
+        choices=("yes", "no"),
+        help="Override git option in cfbs.json",
```

Comment:
Isn't there something like `type=bool` that understands the usual "boolean" values for options and sets the value to `None, True, False` based on it?

Reply:
There is a `BooleanOptionalAction` that accepts `--git` or `--no-git`, but it was not introduced before python 3.9. I could of course subclass `Action` and create my own if that would be desirable.

Reply:
That's probably not necessary.